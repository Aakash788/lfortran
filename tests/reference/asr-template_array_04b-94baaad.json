{
    "basename": "asr-template_array_04b-94baaad",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/template_array_04b.f90",
    "infile_hash": "40518c2e8ea3be4efa6f99a69c7dd47930c37c4384bbb33699c04cd8",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-template_array_04b-94baaad.stderr",
    "stderr_hash": "db72983598d8800b44d8711a6c47a1e72b70ecd3d22ce5ed1ad6cbe2",
    "returncode": 2
}