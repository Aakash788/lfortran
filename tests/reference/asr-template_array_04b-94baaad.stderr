syntax error: Token 'default_behavior' (of type 'identifier') is unexpected here
  --> tests/../integration_tests/template_array_04b.f90:13:18
   |
13 |         requires default_behavior(T)
   |                  ^^^^^^^^^^^^^^^^ 
